<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SysConfigDialog</class>
 <widget class="QDialog" name="SysConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>591</width>
        <height>251</height>
       </rect>
      </property>
      <property name="title">
       <string>报警开关：打开--关闭</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>通道1：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="SwitchButton" name="pushButton_1">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>通道2：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="SwitchButton" name="pushButton_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>通道3：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="SwitchButton" name="pushButton_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>通道4：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="SwitchButton" name="pushButton_4">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>通道5：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="SwitchButton" name="pushButton_5">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>通道6：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="SwitchButton" name="pushButton_6">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>通道7：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="SwitchButton" name="pushButton_7">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>通道8：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="7">
        <widget class="SwitchButton" name="pushButton_8">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>通道9：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="SwitchButton" name="pushButton_9">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>通道10：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="SwitchButton" name="pushButton_10">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>通道11：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="SwitchButton" name="pushButton_11">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="6">
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>通道12：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="7">
        <widget class="SwitchButton" name="pushButton_12">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>通道13：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="SwitchButton" name="pushButton_13">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_16">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>通道14：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="SwitchButton" name="pushButton_14">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLabel" name="label_17">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>通道15：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="SwitchButton" name="pushButton_15">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="6">
        <widget class="QLabel" name="label_18">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>通道16：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="7">
        <widget class="SwitchButton" name="pushButton_16">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>591</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>报警消息设置：</string>
      </property>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>SMTP服务器：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="smtpName">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>20</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>端口号：</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="smtpPort">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>20</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>25</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>20</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>SSL</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="smtpSSL">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>20</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>启用</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>用户名：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="userName">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>50</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>密码：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="password">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>50</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>收件人：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="receivers">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>351</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>主题：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="subject">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>110</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>发送间隔</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="sendTime">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>110</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>112</y>
         <width>54</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>分</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_OK">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>80</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_cancel">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>110</y>
         <width>75</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_test">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>测试</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QPushButton</extends>
   <header location="global">switchbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
